docs/glossary.md

```diff
@@ -49,6 +49,25 @@
    language to analyze the behaviour of the program (e.g. through symbolic
    execution), and uses what it learned to improve the compilation result.
 
+<a name="sort-injection"></a>*sort injection*
+
+1. (noun)
+   A [symbol](#symbol) with the `sortInjection` attribute. The sort injection
+   symbol is used to represent the K sub-sort relation in Kore: K sorts contain
+   symbols and sorts (their sub-sorts), but Kore sorts contain only symbols; the
+   sort injection symbol wraps patterns of a sub-sort so they can be included
+   (_injected_) into the super-sort.
```

Comment:
Is it worth explaining the relation between sort injections and constructors, i.e. that injections behave mostly as constructors (no-junk and no-confusion)? It is contained implicitly in the "K sorts contain symbols and sorts" part, but I'm not sure it's obvious enough.

Reply:
Yes, that's a good idea.